config/custom-environment-variables.json

```diff
@@ -18,17 +18,16 @@
     "googleServiceEmail": "GOOGLE_SERVICE_EMAIL",
     "googleServiceKey": "GOOGLE_SERVICE_KEY",
     "googleServiceKeyId": "GOOGLE_SERVICE_KEY_ID",
-    "prevoteFolderIds": {
-      "Philadelphia": "PHILADELPHIA_PREVOTE_FOLDER_ID",
-      "Boston": "BOSTON_PREVOTE_FOLDER_ID",
-      "New York City": "NEW_YORK_PREVOTE_FOLDER_ID",
-      "San Francisco": "SAN_FRANCISCO_PREVOTE_FOLDER_ID"
+    "teamFolderIds": {
+      "Philadelphia": "PHILADELPHIA_FOLDER_ID",
+      "Boston": "BOSTON_FOLDER_ID",
+      "New York City": "NEW_YORK_CITY_FOLDER_ID",
+      "San Francisco": "SAN_FRANCISCO_FOLDER_ID"
     },
-    "snapshotFolderIds": {
-      "Philadelphia": "PHILADELPHIA_SNAPSHOT_FOLDER_ID",
-      "Boston": "BOSTON_SNAPSHOT_FOLDER_ID",
-      "New York City": "NEW_YORK_SNAPSHOT_FOLDER_ID",
-      "San Francisco": "SAN_FRANCISCO_SNAPSHOT_FOLDER_ID"
+    "templateIds": {
+      "external-snapshot": "EXTERNAL_SNAPSHOT_TEMPLATE_ID",
```

Comment:
I think this is fine to leave in `production.json` instead of configuring through environment variables, now that I think of it. The templates aren't public anyway.

Reply:
I think it's better to be safe. I don't think it's worth to have any info from our drive to be public.